chema-cabeza/Precurso/Calculator-html/script.js

```diff
@@ -0,0 +1,76 @@
+let preValues=[];
+let equalValueFlag = false; // Flag used to determinate what the undo button will do (false: it will delete the last digit/sign introduced / true: It will bring back the previous mathematical function)
+
+function decimal(){
+
```

Comment:
como en los otros proyectos intentar no dejar saltos de linea tras function, for, if, else ...